posts/helpers.py

```diff
@@ -32,9 +32,10 @@ def parse_id_from_url(url):
     Parses a user id from user urls in the form:
     https://example.com/author/f3be7f78-d878-46c5-8513-e9ef346a759d/
     """
-    user_url = url.split('/author/')[-1]
-    user_url = user_url[:-1] if user_url[-1] == '/' else user_url
-    return user_url
+    parsed = urlparse(url)
+    path = parsed.path.strip('/')
```

Comment:
Is this intentional? Don't want us to revert to using something that had issues before

Reply:
Yes, yours and kieters fix breaks on viewing external users. When I reverted to the previous implementation it worked

Reply:
Mike it works  FINE

Reply:
<3